### PR TITLE
Began using Topcoat to pull in Minicon Asset Library from Capital Framework

### PR DESCRIPTION
**Pull request by iuloshi**
Also:
- added `max-width` to `p` tag
- styled Typography page examples

Note: the Minicons Asset Library isn't fulled styled yet. That's in the works.

### CODE REVIEW

**Comment by Scotchester** [comment]
Not sure what happened, but this is recommitting the changes from the last two merge pull requests, which is probably why it can't be automatically merged.

**Comment by iuloshi** [comment]
Hokay, think that fixed it.

File: identity/minicons.md [hunk 1]
```diff
@@ -165,3 +165,10 @@ Padding varies around minicons of various primary shapes. Square, vertically and
 </div>
 
 ## Asset library
+
+{::nomarkdown}
+{% include identity/icons/index.html %}
+{:/nomarkdown}
```
Comment (Scotchester) [inline identity/minicons.md:171]: Shouldn't need the `nomarkdown` around this include statement.

File: front/topdoc-templates/icons/index.jade [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
-				if(pattern.name && pattern.markup)
-					div(class="minicon-assets-div")
-						h6=pattern.name
-						!=pattern.markup
+		h3=component.name
+		each pattern in component.patterns
+			if(pattern.name && pattern.markup)
+				div(class="minicon-assets-div")
+					h6=pattern.name
+					!=pattern.markup
```
Comment (Scotchester) [inline front/topdoc-templates/icons/index.jade:9]: Looks like the indentation got off a little here, but I think Mike will be adjusting this template anyway, so don't worry about it for now.

Comment (iuloshi) [inline front/topdoc-templates/icons/index.jade:9]: We tried indenting `each pattern in component.patterns` but then the subsequent content was nested inside the `h3`.

Comment (Scotchester) [inline front/topdoc-templates/icons/index.jade:9]: The `each pattern` has to be nested under the `if` statement on line 3. If not, that `if` statement has no effect. The `h3` should probably be move between lines 2 and 3, given its content.

File: identity/minicons.md (hunk 1 above)
Comment (iuloshi) [inline identity/minicons.md:171]: Removing the `nomarkdown` made the minicons lose some styling/turn gray... 

screen shot 2014-04-14 at 2 04 47 pm

I copy/pasted that code from the other UI toolkit pages. If it's not hurting, maybe we can keep it in there for now.

File: front/topdoc-templates/icons/index.jade (hunk 2 above)
Comment (iuloshi) [inline front/topdoc-templates/icons/index.jade:9]: Ah, that makes sense.

File: identity/minicons.md (hunk 1 above)
Comment (Scotchester) [inline identity/minicons.md:171]: The color change? Interesting... Yeah, if it's not doing any harm, leave it in. Just noticed the stray `</div>` right below this. Can you remove that?

Comment (iuloshi) [inline identity/minicons.md:171]: Yep just saw that too!

**Pull request merged**